### PR TITLE
Fix the broken link in the code of conduct file

### PR DESCRIPTION
**Pull request by logangamesdaily**

### CODE REVIEW

**Comment by logangamesdaily** [comment]
yes, this is picky, but simply clicking the link does not work. probably because of the trailing “.” but this one works so ¯\\\_(ツ)_/¯

**Comment by Catta1997** [comment]
The problem is not the "." but the problem is that the link is expired. the link should be changed to `https://discord.com/invite/RNCHsQHr3S` instead of `[https://discord.gg/rMv5qxGTGC](https://discord.gg/playcover-871829896492642387)`.

**Comment by Catta1997** [comment]
Same things in [README.md](https://github.com/PlayCover/PlayCover/blob/develop/README.md) file

**Comment by logangamesdaily** [comment]
You are correct, will fix that too.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-https://discord.gg/rMv5qxGTGC.
+[https://discord.gg/rMv5qxGTGC](https://discord.gg/playcover-871829896492642387).
```
Comment (TheMoonThatRises) [inline CODE_OF_CONDUCT.md]: The invite link should be `https://discord.gg/RNCHsQHr3S`, and the url inside the brackets should be the same as the link that the user is directed to.

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@
     <br />
     <a href="https://playcover.github.io/PlayBook">Documentation</a>
     ·
-    <a href="https://discord.gg/rMv5qxGTGC">Discord</a>
+    <a href="https://discord.com/invite/RNCHsQHr3S">Discord</a>
```
Comment (TheMoonThatRises) [inline README.md]: Can you use the `.gg` url for consistency?

**Review by TheMoonThatRises — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (logangamesdaily) [reply CODE_OF_CONDUCT.md]: sorry, just pasted what discord gave me.
weird, will fix

File: README.md (hunk 2 above)
Comment (logangamesdaily) [reply README.md]: yep!

**Review by logangamesdaily — COMMENTED** [review]
Finished

**Review by TheMoonThatRises — APPROVED** [review]

**Pull request merged**